pkg/util/runner.go

```diff
@@ -33,6 +33,7 @@ func (r RealRunner) Run(command string, args ...string) error {
 	fmt.Println(fmt.Sprintf("Provisioning OpenShift via '%s %s'", command, args))
 	cmd := exec.Command(command, args...)
 	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stdout
```

Comment:
@hferentschik That is something we should handle as a separate PR I can remove this bit for this and then we can have separate issue for this.